scopesim_templates/tests/test_micado/test_viking_fields.py

```diff
@@ -26,7 +26,7 @@ def test_load_galaxies_source(self, cat_id):
 
         assert len(gal_src.fields) > 1
         assert len(gal_src.spectra) == 1
-        assert len(gal_src.fields) == len(gal_src._meta_dicts)
+        # assert len(gal_src.fields) == len(gal_src._meta_dicts)
```

Comment:
Why is this now necessary? Feels like a it could potentially be a problem, because IIRC some code relies on the fact that the number of fields and the number of meta dicts is the same

Reply:
`Source._meta_dicts` was removed in [`068330`](https://github.com/AstarVienna/ScopeSim/commit/0683306e9d9fcae86911f42266ebee7fc31f8f14) (with a rather fitting name...) after your comment in https://github.com/AstarVienna/ScopeSim/pull/405#discussion_r1741535428.

Reply:
```suggestion
```
Ah yes. So let's remove it :+1: